index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="stylesheet" href="css/style.css">
+  <title>Document</title>
+</head>
+<body>
+  <header>
+    <nav>
+      <div class="mob-navbar">
+        <a href="#">Welcome</a>
+        <img src="image/Union.png" alt="option">
+      </div>
+    </nav>
+  </header>
+  <main>
+    <section class="title-section">
+      <div class="landing-text">
+        <h1>Hay there. I'm Mariane</h1>
+        <h2>I'm a software developer</h2>
+        <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+          don’t hestiate to contact me.</p>
+      </div>
+      <ul class="social-media-icons">
+        <li>
+          <img class="icon" src="image/twitter.png" alt="Twitter image">
```

Comment:
- [ ] [REQUIRED] Please, add a dead link representing surrounding each `img` tag to make the icons clickable. As requested in project requirements :receipt:.

## example

```html
<ul>
  <li>
    <a href="#">
      <img src="./path" />
    </a>
  </li>
</ul>
```